### PR TITLE
rename test_*pipeline/solid* -> test_*job/op*

### PR DESCRIPTION
**Pull request by smackesey**
## Summary & Motivation

Internal companion PR: https://github.com/dagster-io/internal/pull/5554

Rename test functions containing "pipeline" or "solid" to "job"/"op" equivalent.

## How I Tested These Changes

Existing test suite.

### CODE REVIEW

**Comment by smackesey** [comment]
Current dependencies on/for this PR:
* master
  * **PR #13507** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13507" target="_blank">Graphite</a> 
    * **PR #13482** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13482" target="_blank">Graphite</a> 
    * **PR #13327** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13327" target="_blank">Graphite</a> 
      * **PR #13496** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13496" target="_blank">Graphite</a> 
        * **PR #13981** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13981" target="_blank">Graphite</a> 
          * **PR #13982** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13982" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/13982?utm_source=stack-comment).

**Comment by smackesey** [comment]
Current dependencies on/for this PR:
* master
  * **PR #13507** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13507" target="_blank">Graphite</a> 
    * **PR #13482** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13482" target="_blank">Graphite</a> 
    * **PR #13327** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13327" target="_blank">Graphite</a> 
      * **PR #13496** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13496" target="_blank">Graphite</a> 
        * **PR #13981** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13981" target="_blank">Graphite</a> 
          * **PR #13982** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13982" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/13982?utm_source=stack-comment).

File: python_modules/dagster/dagster_tests/core_tests/execution_tests/test_context.py [hunk 1]
```diff
@@ -20,26 +20,3 @@ def foo():
...
-
-        check.inst(context.run, DagsterRun)
-        assert context.job_name == "foo"
-        assert context.op_config is None
-
-    @job
-    def foo():
-        ctx_op()
-
-    assert foo.execute_in_process().success
```
Comment (dpeng817) [inline python_modules/dagster/dagster_tests/core_tests/execution_tests/test_context.py:45]: is there an equivalent test for OpExecutionContext?

File: python_modules/dagster/dagster_tests/definitions_tests/decorators_tests/test_op.py [hunk 2]
```diff
@@ -409,15 +408,6 @@ def the_out_op():
     assert len(the_op.outs) == 0
 
 
-def test_op():
-    @op
-    def my_op():
-        pass
-
-    assert isinstance(my_op, OpDefinition)
-    execute_op_in_graph(my_op)
```
Comment (dpeng817) [inline python_modules/dagster/dagster_tests/definitions_tests/decorators_tests/test_op.py:418]: I think this is definitely fine to get rid of

**Review by dpeng817 — APPROVED** [review]
one comment

File: python_modules/dagster/dagster_tests/core_tests/execution_tests/test_context.py (hunk 1 above)
Comment (smackesey) [reply python_modules/dagster/dagster_tests/core_tests/execution_tests/test_context.py:45]: yea it's the only other test in the file

**Pull request merged**